app/mailers/student_mailer.rb

```diff
@@ -11,7 +11,7 @@ class StudentMailer < ApplicationMailer
   def draw_invitation(user, college)
     @user = user
     @intent_deadline = user.draw.intent_deadline
-    @college = college
+    @college = college || College.new
```

Comment:
Also, it occurs to me that we don't need to pass the college to the mailer methods at this stage - we can just set it to `College.first || College.new` globally. On the flip side, we'll eventually want to pass in a college (unless we just want to inherit it from the user?) - I don't feel strongly either way so it's your call.